test/kube2e/helper/testserver.go

```diff
@@ -10,74 +10,13 @@ import (
 )
 
 const (
-	defaultTestServerImage = "quay.io/solo-io/testrunner:v1.7.0-beta17"
+	defaultTestServerImage = "gcr.io/solo-test-236622/testrunner:v1.7.0-beta18"
 	TestServerName         = "testserver"
 	TestServerPort         = 1234
 
 	// This response is given by the testserver when the SimpleServer is started
-	SimpleHttpResponse = `<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"><html>
-<title>Directory listing for /</title>
```

Comment:
purged dom as multiarch image was returning slightly diff dirs for amd64, arm64. also it had symlinks in dir path 

if still needed I can work on keeping the dom and updating test code to ensure assertions work

Reply:
It looks like this is used to match what is returned from the testserver. Can we expand this to at least include the title to make sure we are matching on more than the DOCTYPE?

Reply:
yeah lemme add back title